README.md

```diff
@@ -1,25 +1,33 @@
-<a href="url"><img src="docs/images/magma-logo-purple.svg" width="300" ></a>
+<h1 align="center">
+  <a href="https://magmacore.org/"><img src="docs/images/magma-logo-purple.svg" width="500" ></a>
+  <p align="center"><a href="https://charmhub.io/magma-orc8r"><img src="https://charmhub.io/magma-orc8r/badge.svg" alt="magma orchestrator badge"/></p>
+  <p align="center"><a href="https://charmhub.io/magma-access-gateway-operator"><img src="https://charmhub.io/magma-access-gateway-operator/badge.svg" alt="access gateway badge"/></p>
+</h1>
 
-**Magma** is an open source 4G and 5G core network. Use it to operate your own private mobile 
+![Get it from Charmhub](https://charmhub.io/static/images/badges/en/charmhub-black.svg)
+
+[Don't have Juju installed?](https://snapcraft.io/juju)
```

Comment:
I don't think this is necessary.
We have some references to juju further in the text.

Reply:
Removed.